code/game/objects/structures/mouse_cage.dm

```diff
@@ -71,6 +71,7 @@
 /obj/item/critter_cage/lock_atom(var/atom/movable/AM)
 	. = ..()
 	if (.)
+		AM.plane = MOB_PLANE
```

Comment:
Is there something in the code that reverts the plane back?

Reply:
I considered that it wasn't a big deal since only cockroaches spawn by default at a lower plane but I've added a revert to the initial value in unlock_atom.